addons/spectator/functions/fnc_compat_spectatorBI.sqf

```diff
@@ -22,6 +22,10 @@ if !(QUOTE(ADDON) in _templates) exitWith {};
 // Start our spectator
 [true] call FUNC(setSpectator);
 
+// Delete the seagull that spawns (not actually the player, a CfgNonAIVehicles object)
+// This could delete seagulls created by a wildlife module (TODO: Find a more accurate method of getting the exact seagull)
```

Comment:
I'll do some more testing related to this. if so then I could have the template take care of the seagull while this display XEH handles replacing BI's spectator